lib/ts/recipe/multifactorauth/recipe.ts

```diff
@@ -69,8 +71,16 @@ export default class Recipe extends RecipeModule {
         this.isInServerlessEnv = isInServerlessEnv;
 
         {
-            let builder = new OverrideableBuilder(RecipeImplementation(this));
+            let originalImpl = RecipeImplementation(this);
+            let builder = new OverrideableBuilder(originalImpl);
             this.recipeInterfaceImpl = builder.override(this.config.override.functions).build();
+
+            // @ts-ignore
+            let layers = builder.layers;
+
+            if (Object.keys(layers[1]).includes("getMFARequirementsForAuth")) {
+                this.isGetMfaRequirementsForAuthOverridden = true;
+            }
```

Comment:
just check that the override function is provided and nothing else is needed

Reply:
Done